### PR TITLE
Updating enums and function_addon

### LINKED ISSUE #388: Populate nidcpower functions_addon.py
We are generating using untouched extracted metadata. API isn't usable without adding extra metadata via functions_addon.py.

### PR DESCRIPTION
**Pull request by injaleea**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

[X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

### What does this Pull Request accomplish?

Adding enums MeasurementTypes, OutputStates and updated function_addon to overwrite to these enums
Added System tests 
- test_measure
- test_query_output_state

### List issues fixed by this Pull Request below, if any.

* Fix #388 

### What testing has been done?

tox and system tests

### CODE REVIEW

**Comment by Fladolcetta** [comment]
One comment, can you add the commented out enum tests to this review and remove them from #449 ?

Also, where did you get documentation, name and value of the added enums?

File: src/nidcpower/metadata/enums.py [hunk 1]
```diff
@@ -566,4 +566,40 @@
...
+            {
+                'name': 'MEASURE_VOLTAGE',
+                'value': 1,
+                'documentation': {
+                    'description': 'The device measures voltage.',
+                },
+            },
+            {
+                'name': 'MEASURE_CURRENT',
+                'value': 0,
```
Comment (Fladolcetta) [inline src/nidcpower/metadata/enums.py]: Can we move this above measure voltage? That way 0 comes before 1?

**Comment by injaleea** [comment]
Documentation and values are taken from session.py

**Comment by injaleea** [comment]
Yes, I will be updating internal files once @Fladolcetta's changes goes into it, so that I have a reference reference to update.

File: src/nidcpower/metadata/enums.py [hunk 2]
```diff
@@ -566,4 +566,40 @@
             },
         ],
     },
+    'MeasurementTypes': {
+        'values': [
+            {
+                'name': 'MEASURE_CURRENT',
+                'value': 0,
+                'documentation': {
+                    'description': 'The device measures current.',
```
Comment (marcoskirsch) [inline src/nidcpower/metadata/enums.py]: Where does this documentation come from? Was this file generated locally using the internal tools?

Comment (injaleea) [reply src/nidcpower/metadata/enums.py]: Documentations are taken from session.py . Updated internal files to generate with same documentation.

**Pull request merged**